gdcdictionary/schemas/clinical_trial.yaml

```diff
@@ -0,0 +1,342 @@
+  - data_category
+  - data_type
+
+uniqueKeys:
+  - [id]
+  - [project_id, submitter_id]
+
+# Case properties
+properties:
+
+  $ref: "_definitions.yaml#/data_file_properties"
+
+  type:
+    enum: [ "clinical_trial" ]
+
+  data_category:
+    term:
+      $ref: "_terms.yaml#/data_category"
+    type: string
+
+  data_type:
+    term:
+      $ref: "_terms.yaml#/data_type"
+    type: string
+
+  data_format:
+    term:
+      $ref: "_terms.yaml#/data_format"
+    type: string
+
+  submitted_date:
+    description: >
+      First Submitted Date
+    type: string
+
+  first_posted_date:
+    description: >
+      First Posted Date
+    type: string
+
+  last_update_posted_date:
+    description: >
+      Last Update Posted Date
+    type: string
+
+  actual_study_start_date:
+    description: >
+      Actual Study Start Date
+    type: string
+
+  actual_primary_completion_date:
+    description: >
+      Actual Primary Completion Date
+    type: string
+
+  title:
+    description: >
+      Name for this study (Brief Title)
+    type: string
+
+  official_title:
+    description: >
+      Official name for this study (Official Title)
+    type: string
+
+  description:
+    description: >
+      What this is study doing (Brief Summary)
+    type: string
+
+  detailed_description:
+    description: >
+      Detailed description of what this study is doing
+    type: string
+
+  category:
+    description: >
+      Classifications for the study (Study Type)
+    enum:
+      - "Investigational"
+      - "Observational"
+
+  phase:
+    description: >
+      Study Phase
+    enum:
+      - "n-a"
+      - "early-phase-1"
+      - "phase-1"
+      - "phase-1-phase-2"
+      - "phase-2"
+      - "phase-2-phase-3"
+      - "phase-3"
+      - "phase-4"
+  
+  study_design_allocation:
+    description: >
+      Study Design: Allocation
+    type: string
+
+  study_design_intervention:
+    description: >
+      Study Design: Intervention Model
+    type: string
+
+  study_design_masking:
+    description: >
+      Study Design: Masking
+    type: string
+
+  study_design_primary_purpose:
+    description: >
+      Study Design: Primary Purpose
+    type: string
+
+  condition:
+    description: >
+      Study Design: Primary Purpose
+    enum:
+      - "COVID-19"
+      - "Other"
+    
+  focus:
+    description: >
+      Codes for medications, devices and other interventions. (Intervention)
+    enum:
+      - "Drug"
+      - "Other"
+
+  focus_spec:
+    description: >
+      Specify the codes for medications, devices and other interventions.
+    enum:
+      - "Placebo"
+      - "Remdesivir"
+    
+  arm_name:
+    description: >
+      Label for study arm
+    enum:
+      - "Placebo"
+      - "Remdesivir"
+
+  arm_type:
+    description: >
+      Categorization of study arm
+    enum:
+      - "Active Comparator"
+      - "Placebo Comparator"
+      - "Experimental"
+
+  arm_description:
+    description: >
+      Short explanation of study path
```

Comment:
Unless this came from FHIR, update to "Short explanation of study arm"

Reply:
It is from FHIR